stories/action-sheet/action-sheet.stories.js

```diff
@@ -0,0 +1,200 @@
+                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left"
+                        onclick="onPopoverClick('actionSheetDesktop');">
+                    <span class="fd-button__text">Decide Later</span>
+                </button>
+            </li>
+        </ul>
+    </div>
+</div>
+<div style="height: 300px"></div>
+`;
+
+actionSheetDesktop.parameters = {
+    docs: {
+        iframeHeight: 300
+    }
+};
+
+
+export const actionSheetTablet = () => `    
+<div class="fd-popover">
+    <div class="fd-popover__control">
+        <button class="fd-button fd-button--transparent" 
+                aria-label="More Options" aria-controls="actionSheetTablet" aria-expanded="false" 
+                aria-haspopup="true" onclick="onPopoverClick('actionSheetTablet');">
+            <i class="sap-icon--settings"></i>
+        </button>
+    </div>
+    <div class="fd-popover__body" aria-hidden="true" id="actionSheetTablet">
+        <ul class="fd-action-sheet" role="list" aria-label="List of contextual options">
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--attachment"></i>
+                    <span class="fd-button__text">Upload file</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--add"></i>
+                    <span class="fd-button__text">Assign user</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--print"></i>
+                    <span class="fd-button__text">Print</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--download"></i>
+                    <span class="fd-button__text">Download</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--transparent fd-button--text-alignment-left"
+                        onclick="onPopoverClick('actionSheetTablet');">
+                    <span class="fd-button__text">Decide Later</span>
+                </button>
+            </li>
+        </ul>
+    </div>
+</div>
+<div style="height: 300px"></div>
+`;
+
+actionSheetTablet.parameters = {
```

Comment:
(If above is makes sense)

For tablet screens, add the `id="actionSheetTablet"` to the `fd-popover__body` component.